src/components/OrderableList/OrderableList.tsx

```diff
@@ -58,7 +59,7 @@ export const OrderableList = <T extends object>({
                             key={`orderable-list-item-${index}-before`}
                             data={{
                                 targetItem: item,
-                                position: DropZonePosition.Before,
+                                position: 'before' as const,
```

Comment:
Why the `as const` and not `as CollisionPosition`

Reply:
As it is an untyped object value, and value, in this case, for the position is 'before' so `const` is correct. If I were to give a type to the object I'm passing then it would be `CollisionPosition` and there would be no reason for me to do `as` at all.